.buildkite/pipeline_cpu_template.py

```diff
@@ -70,7 +70,6 @@ class BkStep(str, Enum):
             "m5n.metal",
             "m6i.metal",
             "m6a.metal",
-            "m7a.metal-48xl",
```

Comment:
Note that we remove this because m7a.metal does not support any existing CPU templates. Without this removal, we'll get an error in the artifact upload step.